Project-2 Finn

```diff
@@ -0,0 +1,258 @@
+```
+I have a mermaid diagram for this one as well, but can't figure out how to insert it.
+
+ΔI1 ≡ {John Beverley, PHI 697, PHI 329}
+
+Named IndividualsI:
+John BeverleyI ≡ B
+PHI 697I ≡ 697
+PHI 329I ≡ 329
+
+Role Assignments:
+t ≡ {(B, 697), (B, 329)}
+
+ΔI2 ≡ {John Beverley, PHI 697, PHI 329}
+
+Named IndividualsI:
+John BeverleyI2 ≡ B2
+PHI2 697I ≡ 697-2
+PHI 329I ≡ 329-2
+
+Role Assignments:
+t2 ≡ {(B2, 697-2),  (B, 329-2)}
+
+Bisimulation:
+
+ρ ≡ {(B, B2), (697, 697-2), (329, 329-2)}
+
+So B is bisimilar to B2. The ALCN definitions are
+
+≥2 ∃m.⊤
+in I1
+
+≥2 ∃m2.⊤
+in I2
+
+```
+
+6. Explain the difference - using natural language - between the description logic expressions:
+  ```
+  (a) ∃r.C and ∀r.C
+  -The first is talking about anything that r's something that's a C, while the second is talking about everything that r's everything that's a C.
+  (b) ∃r-.C and ∀r-.C
+  -The first is talking about something that's C that r's everything, while the second is talking about everything that's C that r's everything.
+  (c) <=nr and <=nr.C
+  -<=nr means there is a number of r-fillers that's less than or equal to some amount, and <=nr.C means there is a number of r-fillers that's less than or equal to some amount, and they are Cs.
+  (d) ∃r-.C and ∃r-.{a} 
+  -The first says there's something that's a member of a concept that is in an r relation to everything, while the second says that something that's a concept is in an r relation to everything.
+```
+
+7. There is a delightfully helpful subreddit called "ELI5" which stands for something like "explain it like I'm 5" where users post conceptually challenging questions and other users attempt to provide explanations in simple, jargon-free, terms that presumably a 5 year-old could understand. Using this as a model, explain the _finite model property_. Be sure to provide a simple example and explain when the property might be important, and when it is not so important. 
+```
+Say that I want to show that Ottawa is the capital of Canada. I could open a map that shows each country's capital to demonstrate this. If that doesn't convince you, we could ask someone from Canada, or even go to Canada ourselves. But to show this, I don't need to know the capital of Australia, or how an internal combustion engine works. There's lots of facts that I don't need to use to show that Canada's capital is Ottawa. If something has a finite model property, then you don't need to use every fact to show it's true. This is important, because computers also try to show that some things are true, but without the finite model property, they would be stuck calculating forever, since they would need to use all the facts.
```

Comment:
Close, you don't need to use _uncountably_ many facts to show something is true.